mud-contracts/smart-object-framework-v2/src/interfaces/ITags.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { ResourceId } from "@latticexyz/world/src/WorldResourceId.sol";
+import { Id } from "../libs/Id.sol";
+
+/**
+ * @title ITags
+ * @dev An interface for the Tags System functionality
+ */
+interface ITags {
+	function setSystemTag(Id classId, Id systemTagId) external;
```

Comment:
Nitpick: Strange indent

Reply:
even I found this strange behaviour, but apparently its only in github UI. Local it should be fine

Reply:
Linting done in  commit 2350b72266add31123d779ce4c427a67882f4ab8